tests/Makefile

```diff
@@ -0,0 +1,8 @@
+permissions:
+	./stacker.yaml.sh | stacker build -
+
+test: permissions
```

Comment:
We already have a `test-functional` target in the top level Makefile. Maybe move this there? https://github.com/remind101/stacker/blob/master/Makefile#L13

Reply:
Yeah